src/main.ts

```diff
@@ -1 +1,3 @@
-console.log("Hello world!")
+let yourName = "John";
```

Comment:
Codacy has a fix for the issue: ['yourName' is never reassigned. Use 'const' instead.](https://app.codacy.com/gh/lolgab/eslint-example/pullRequest?prid=6956677)
```suggestion
const yourName = "John";
```